### PR TITLE
Initial draft of "This Year In Monero - 2021"

### PR DESCRIPTION
**Pull request by sethsimmons**
This is my first post here, but I think it should be pretty much good to go!

Please let me know if you have any further suggestions or any changes that need to be made.

Thanks!

### CODE REVIEW

**Comment by sethsimmons** [comment]
Netlify preview looks good to me now that image tags are fixed.

Should be ready for review/merge!

File: _posts/2021-04-18-this-year-in-monero.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+## Twitter
+
+***04/2019 through 04/2020:***
+
+- Monero's main Twitter account, [@monero](https://twitter.com/monero) gained ***3,502*** followers.
+- The @monero Twitter account went from ***317,010*** to ***320,512*** followers.
+
+***04/2020 through 04/2021:***
+
+- Monero's main subreddit, [@monero](https://twitter.com/monero) gained ***69,209*** followers.
```
Comment (erciccione) [inline _posts/2021-04-18-this-year-in-monero.md:178]: You meant "Twitter account"

**Review by erciccione — CHANGES_REQUESTED** [review]
Nice post. Two requests:

- Please run the png images through a png compressor like tinypng.com and then resubmit them. It will save a lot of space. For example `monero-hashrate-2021.png` could be shrinked from 330kB to 94kB, a ~72% saving.

- This article is taken from your blog, please add a disclaimer at the bottom of the post with a link to the original article (https://sethsimmons.me/posts/this-year-in-monero-2021/). It will avoid future confusions.

File: _posts/2021-04-18-this-year-in-monero.md (hunk 1 above)
Comment (sethsimmons) [reply _posts/2021-04-18-this-year-in-monero.md:178]: Fixed.

**Comment by sethsimmons** [comment]
> Nice post. Two requests:
> 
>     * Please run the png images through a png compressor like tinypng.com and then resubmit them. It will save a lot of space. For example `monero-hashrate-2021.png` could be shrinked from 330kB to 94kB, a ~72% saving.
> 
>     * This article is taken from your blog, please add a disclaimer at the bottom of the post with a link to the original article (https://sethsimmons.me/posts/this-year-in-monero-2021/). It will avoid future confusions.

All changes made and pushed. Will validate that they look alright once the CI runs and provides a preview.

Thanks for the suggestions!

**Comment by sethsimmons** [comment]
For future reference https://github.com/kornelski/pngquant was a great tool for this, ended up using that for my own blog post as a test and was able to shrink them even further using:

`pngquant *.png --strip --skip-if-larger --quality=65-80 --verbose --ext=.png --force`

I'll use that in future posts.

**Review by erciccione — COMMENTED** [review]
> For future reference https://github.com/kornelski/pngquant was a great tool for this

Good to know. Thanks for the link :)

Just one more thing and then this will be good to go: could you update the file name with today's date? People using RSS feeds will be confused if they receive a feed for a post that appear to have been published a week ago. Nothing important, but since it's a quick fix, better do it.

**Comment by sethsimmons** [comment]
> > For future reference https://github.com/kornelski/pngquant was a great tool for this
> 
> 
> 
> Good to know. Thanks for the link :)
> 
> 
> 
> Just one more thing and then this will be good to go: could you update the file name with today's date? People using RSS feeds will be confused if they receive a feed for a post that appear to have been published a week ago. Nothing important, but since it's a quick fix, better do it.

Done! Also added a minor note on the miner count numbers that I had added to my own blog in the meantime.

**Review by erciccione — APPROVED** [review]
Needs to be squashed.

**Review by erciccione — APPROVED** [review]

**Pull request merged**